config/environments/production.rb

```diff
@@ -78,6 +78,9 @@
     enable_starttls_auto: true
   }
 
+  hostname = "#{ENV.fetch('HEROKU_APP_NAME')}.herokuapp.com"
```

Comment:
Could it make sense to do the following?:
```suggestion
  hostname = "#{ENV.fetch('HEROKU_APP_NAME', "PROJECT_DOMAIN")}.herokuapp.com"
```

Reply:
```suggestion
  hostname = ENV['DOMAIN_NAME']
  hostname ||= "#{ENV.fetch('HEROKU_APP_NAME')}.herokuapp.com"
```